scss/variables/components/_alerts.scss

```diff
@@ -15,7 +15,9 @@
 //
 // Styleguide: Variables.Components.Alerts.decanter-custom-alerts
 //
-$decanter-custom-alerts: () !default;
+$decanter-custom-alerts: (
+    sample: rgba(125, 125, 125, 0.5), // sample default value
```

Comment:
Why is there a default value if it is only a sample? This would be better as a comment if not intended for use.

Reply:
With Neat 3.0, we need at least one value in there, or we get an error.